ocp_resources/plan.py

```diff
@@ -58,8 +62,8 @@ def __init__(
         warm_migration: bool = False,
         pre_hook_name: str | None = None,
         pre_hook_namespace: str | None = None,
-        after_hook_name: str | None = None,
-        after_hook_namespace: str | None = None,
+        post_hook_name: str | None = None,
```

Comment:
this is a breaking change
you can add the new args but not drop the old ones (and use where needed) and add deprecation warning on the old ones.